docs/index.md

```diff
@@ -1,7 +1,7 @@
 # Shopsys Framework Knowledge Base
 
 ## Introduction
-* [Installation Using Docker](installation/installation-using-docker.md) and [Native Installation](installation/native-installation.md)
+* [Installation Using Docker](installation/installation-using-docker.md) and [Native Installation](installation/native-installation.md) and [Installation Using Docker on Production Server](installation/installation-using-docker-on-production-server.md)
```

Comment:
The line is quite long. I would prefer a new list item for `Installation Using Docker on Production Server`